### PR TITLE
Feature/regions routes

### PR DESCRIPTION
**Pull request by OrBin**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,6 +1,18 @@
+const categoriesRoutes = require('./categories');
+const contentsRoutes = require('./contents');
+const regionsRoutes = require('./regions');
 const usersRoutes = require('./users');
+const volunteeringEventsRoutes = require('./volunteering_events');
+const volunteeringOffersRoutes = require('./volunteering_offers');
+const volunteeringRequestsRoutes = require('./volunteering_requests');
 
 module.exports = (router) => {
   router.prefix('/v1');
```
Comment (bsuro10) [inline routes/index.js]: I don't familiar with how to work with api versions.
I recommend using just /api prefix or learn how to use api versions.

File: routes/index.js [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+  router.use('/categories', categoriesRoutes);
+  router.use('/contents', contentsRoutes);
+  router.use('/regions', regionsRoutes);
   router.use('/users', usersRoutes);
+  router.use('/volunteering_events', volunteeringEventsRoutes);
+  router.use('/volunteering_offers', volunteeringOffersRoutes);
+  router.use('/volunteering_requests', volunteeringRequestsRoutes);
```
Comment (bsuro10) [inline routes/index.js]: Shouldn't it be with '-'? 
/volunteering-requests?
/volunteering-events?
/volunteering-offers?

Comment (bsuro10) [reply routes/index.js]: Same for the files' names

**Review by bsuro10 — CHANGES_REQUESTED** [review]

File: routes/index.js (hunk 1 above)
Comment (OrBin) [reply routes/index.js]: I think /api prefix should be used when deploying  only. About the version prefix, I'm not sure but I think it's OK for now.

File: routes/index.js (hunk 2 above)
Comment (OrBin) [reply routes/index.js]: You are right about the URLs, but the file names should use underscores.

File: routes/index.js (hunk 1 above)
Comment (bsuro10) [reply routes/index.js]: There is no point to do so, if we dont work with api versions, we dont need to use 'v1'.
Changing later just point that we won't use api versions.

Remember that if we change the prefix to 'v2' then all the API's will be /v2,
I don't think it's the behavior of api versions

Comment (OrBin) [reply routes/index.js]: OK, you are right. I'm removing it then, and if we need it I guess we can do it via k8s.

**Review by bsuro10 — APPROVED** [review]

**Pull request merged**